docs/reference/pebble-help.md

```diff
@@ -0,0 +1,25 @@
+(reference_pebble_help_command)=
+# Pebble help command
```

Comment:
Can we keep the article title as "Pebble help command" but only show "Help command" in the sidecar table of contents? It seems unnecessary to have "Pebble" there.